bitcoin/src/blockdata/script/mod.rs

```diff
@@ -662,22 +640,14 @@ pub(super) fn bytes_to_asm_fmt(script: &[u8], f: &mut dyn fmt::Write) -> fmt::Re
     // This has to be a macro because it needs to break the loop
     macro_rules! read_push_data_len {
         ($iter:expr, $len:literal, $formatter:expr) => {
-            match read_uint_iter($iter, $len) {
+            match read_uint::read_uint_iter($iter, $len) {
                 Ok(n) => {
                     n
                 },
-                Err(UintError::EarlyEndOfScript) => {
+                Err(_) => {
                     $formatter.write_str("<unexpected end>")?;
                     break;
                 }
-                // We got the data in a slice which implies it being shorter than `usize::MAX`
-                // So if we got overflow, we can confidently say the number is higher than length of
-                // the slice even though we don't know the exact number. This implies attempt to push
-                // past end.
-                Err(UintError::NumericOverflow) => {
-                    $formatter.write_str("<push past end>")?;
-                    break;
-                }
```

Comment:
FTR this would be needed again if we ever decided to support 16-bit architectures at least when it comes to `script::Instructions`.